code/src/Styles/Form.css

```diff
@@ -0,0 +1,162 @@
+h1 {
+	font-size: 26px;
+	text-align: center;
+	margin-bottom: 20px;
+}
+h2 {
+	font-size: 22px;
+	margin-bottom: 20px;
+}
+
+h3 {
+	font-size: 20px;
+	margin: 20px 0;
+}
+
+h4 {
+	font-size: 24px;
+	font-weight: bold;
+	margin-bottom: 20px;
+	text-align: center;
+}
+
+.content-container {
+	display: flex;
+	flex-direction: column;
+}
+
+form {
+	display: flex;
+	align-items: center;
+	flex-direction: column;
+	justify-content: space-evenly;
+	width: 100%;
+	height: 80%;
+}
+
+.form-wrapper {
+	background-image: linear-gradient(
+			to bottom,
+			rgba(0, 0, 0, 0.4),
+			rgba(0, 0, 0, 0.3)
+		),
+		url('../Images/season-unsplash.jpg');
+	background-size: cover;
+	background-position-x: center;
+	background-position-y: 45%;
+	height: 100vh;
+	width: 100vw;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+
+.introduction {
+	padding: 0 10px;
+	height: 100vh;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+}
+
+.navigation-start,
+.navigation-reset {
+	display: flex;
+	justify-content: center;
+}
+
+.question-container,
+.reset-wrapper {
+	padding: 0 10px;
+	width: 100%;
+	color: #fdf7ef;
+	display: flex;
+	flex-direction: column;
+	height: 70%;
+	justify-content: center;
+}
+
+/*Styling for text input*/
```

Comment:
These comments are neat, making it easy to read the css/-file!

Reply:
Thank you! :)